read-excel/book.hpp

```diff
@@ -175,13 +187,15 @@ Book::sheet( size_t index ) const
 }
 
 inline void
-Book::loadBook( const std::string & fileName )
+Book::loadBook( std::istream & fileStream, const std::string& fileName )
 {
 	try {
 		clear();
 
-		CompoundFile::File file( fileName );
-		auto stream = file.stream( file.directory( L"Workbook" ) );
+		CompoundFile::File file( fileStream, fileName);
+		auto stream = file.stream( 
+			file.hasDirectory( L"Workbook" ) ? file.directory( L"Workbook" ) 

```

Comment:
Point me to BIFF8 standard where `Book` is standardized in BIFF8, or remove this selection. This library supports BIFF8 only.

Reply:
ok, will be removed. ( But catdoc does the same here - https://github.com/petewarden/catdoc/blob/master/src/xls2csv.c#L166, so maybe this file isn't the one )

Reply:
Done

Reply:
Ok, I will approve checking for `Workbook` or `Book` to load.